### PR TITLE
change ConfigMap to ConfigMapGenerator

### PR DESCRIPTION
**Pull request by Liujingfang1**

### CODE REVIEW

**Comment by Liujingfang1** [comment]
> LGTM. We will have to update examples as well.

I will update examples, no worries.

File: pkg/kinflate/examples/simple/instances/exampleinstance/Kube-manifest.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ bases:
 patches:
 - deployment/deployment.yaml
 #There could also be configmaps in Base, which would make these overlays
-configmaps:
+configmapGenerators:
```
Comment (monopole) [inline pkg/kinflate/examples/simple/instances/exampleinstance/Kube-manifest.yaml]: configMapGenerator    :)

**Comment by Liujingfang1** [comment]
Change `ConfigMap` to `ConfigMapArg`.
Change `ConfigMaps` to `ConfigMapGenerator`.

**Comment by Liujingfang1** [comment]
Updated `SecretGenerator` in a consistent way, PTAL.

**Pull request merged**